tests/test_photoCalib.py

```diff
@@ -626,6 +626,30 @@ def testCreateFromMetadataRaisesNoKey(self):
         result = lsst.afw.image.makePhotoCalib(metadata)
         self.assertIsNone(result)
 
+    def testCreateFromFluxMag0(self):
+        """Test creating from the Calib-style fluxMag0/fluxMag0Err values."""
+        fluxMag0 = 12345
+        fluxMag0Err = 67890
+
+        referenceFlux = (0*u.ABmag).to_value(u.nJy)
+        calibrationErr = referenceFlux*fluxMag0Err/fluxMag0**2
+
+        # create with all zeros
+        photoCalib = lsst.afw.image.makePhotoCalib(0, 0)
+        self.assertEqual(photoCalib.getInstFluxAtZeroMagnitude(), 0)
+        self.assertEqual(photoCalib.getCalibrationMean(), np.inf)
+        self.assertTrue(np.isnan(photoCalib.getCalibrationErr()))
+
+        # create with non-zero fluxMag0, but zero err
+        photoCalib = lsst.afw.image.makePhotoCalib(fluxMag0, 0)
+        self.assertEqual(photoCalib.getInstFluxAtZeroMagnitude(), fluxMag0)
+        self.assertEqual(photoCalib.getCalibrationErr(), 0.0)
+
+        # create with non-zero fluxMag0 and err
+        photoCalib = lsst.afw.image.makePhotoCalib(fluxMag0, fluxMag0Err)
+        self.assertEqual(photoCalib.getInstFluxAtZeroMagnitude(), fluxMag0)
+        self.assertEqual(photoCalib.getCalibrationErr(), calibrationErr)
```

Comment:
I don't really like the fact that this test copies the formula you used in the implementation, it seems prone to human error. Could you write something like the `getInstFluxAtZeroMagnitude` tests, where you check that the resulting `PhotoCalib` behaves as implied by the definition of `fluxMag0Err`?

Reply:
If `getInstFluxAtZeroMagnitude` is correct, then the PhotoCalib will be correct (assuming that its tests itself pass). I think your suggestion would require copying the code from Calib over to here to compare it?

Reply:
No, I meant something like trying to use the `PhotoCalib` to convert a specific flux (error?) where you know the right answer. Duplicating/sharing code between the system and its tests is what I'd like to avoid.

Reply:
One problem is that the old Calib was incorrect at the ~1e-5 level (the reference flux was wrong). I don't know that doing the (correct) calculation and putting that number in gains us anything vs. having the math here.